migration_original/ODS1Stage/tables/Base/ClientProductToCallCenter/spu_translated_ClientProductToCallCenter.sql

```diff
@@ -87,7 +67,8 @@ merge_statement := ' merge into base.clientproducttocallcenter as target using
 ---------------------------------------------------------
 
 if (is_full) then
-    truncate table Base.ClientProductToCallCenter;
+    delete from base.clientproducttocallcenter
```

Comment:
doing this and truncating the table is the same because you only insert records where the callcenterid has this value so you end up deleting the whole thing and is the same as truncating

Reply:
It is not the same because of the legacy records that do not correspond to this ID.